spring-integration-rsocket/src/test/java/org/springframework/integration/rsocket/outbound/RSocketOutboundGatewayIntegrationTests.java

```diff
@@ -74,7 +74,7 @@
 @DirtiesContext
 public class RSocketOutboundGatewayIntegrationTests {
 
-	private static final int PORT = SocketUtils.findAvailableTcpPort();
+	private static int PORT;
```

Comment:
This should no longer be static.

Reply:
??? It is initialized from the `static @BeforeAll` and it won't be available in the `ClientConfig` below which has to be a `static` as well.

Reply:
Sorry; I meant it should no longer be upper case.